### DISCUSSION TITLE
some questions on .cap and .scc

### DISCUSSION THREAD

**Discussion by PreetM07** [SubtitleEdit/subtitleedit#4975] [Q&A] [answered]
I wanted to check if there is any documentation on .scc and .cap formats available?
A quick question regarding this is if .scc and .cap can be made human readable somehow? What encoding they use? If .scc and .cap are same in all the subtitle tools?

**Comment by niksedk** [comment, marked as answer]
Read more about scc here: http://www.theneitherworld.com/mcpoodle/SCC_TOOLS/DOCS/SCC_FORMAT.HTML

SE support multiple formats with the extension ".cap"... but all binary.

SCC and binary formats are not very human readable.

**Reply by PreetM07** [reply]
This is well documented for scc. Thanks.

Please let me know if there is anything for cap.

**Comment by niksedk** [comment]
I had to reverse engineer most binary formats - takes a lot of time, but it's fun.
